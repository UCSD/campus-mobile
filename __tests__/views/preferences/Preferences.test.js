import 'react-native'
import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

// Component to be tested
import PreferencesView from '../../../app/views/preferences/Preferences'

Enzyme.configure({ adapter: new Adapter() })

// Mock props passed down from state
const initialState = {

}

// Set up component to be rendered
function setup(props) {
	return shallow(<PreferencesView {...props} />)
}

test('renders without crashing', () => {
	const tree = setup(initialState)
	expect(tree).toMatchSnapshot()
})
