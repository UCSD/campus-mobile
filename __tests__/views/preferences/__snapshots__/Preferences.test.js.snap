// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<ScrollView>
  <Connect(UserAccount) />
  <withNavigation(PreferencesItem)
    icon="bell-o"
    iconPack="FontAwesome"
    link="Notifications"
    linkType="internal"
    title="Notifications"
  />
  <withNavigation(PreferencesItem)
    icon="cards-variant"
    iconPack="MaterialCommunityIcons"
    link="CardPreferences"
    linkType="internal"
    title="Cards"
  />
  <withNavigation(PreferencesItem)
    icon="new-message"
    iconPack="Entypo"
    link="Feedback"
    linkType="internal"
    title="Feedback"
  />
  <withNavigation(PreferencesItem)
    icon="lock"
    iconPack="Feather"
    link="https://mobile.ucsd.edu/privacy-policy.html"
    linkType="external"
    title="Privacy Policy"
  />
</ScrollView>
`;
